css/css-fonts/variations/font-variation-settings-inherit.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Testing the inheritance of the font-variation-settings property</title>
+    <link rel="help" href="https://www.w3.org/TR/css-fonts-4/#propdef-font-variation-settings" />
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+    <div id="inheritanceTestParent" style="font-variation-settings: 'xxxx' 1, 'xxxx' 2;">
+        <span id="inheritanceTestChildInherited">Abc</span>
+        <span id="inheritanceTestChildOverride" style="font-variation-settings: 'cccc' 1, 'bbbb' 2, 'aaaa' 3, 'aaaa' calc(2 + 2);">Abc</span>
+    </div>
+    <script>
+
+        // Verify computed inheritance of nested elements.
+        var elementParent = document.getElementById("inheritanceTestParent");
+        var elementChildInherited = document.getElementById("inheritanceTestChildInherited");
+        var elementChildOverride = document.getElementById("inheritanceTestChildOverride");
+
+        var parentValue = "'yyyy' 1, 'yyyy' 2";
+        elementParent.style.fontVariationSettings = parentValue;
+
+        test(() => {
+            var actualValue = window.getComputedStyle(elementParent).fontVariationSettings;
+            // The following strict test is subject to debate; softening for now:
```

Comment:
Filed https://github.com/w3c/csswg-drafts/issues/2288

Reply:
@drott did you want to hold off on approving all tests while that issue gets discussed, or approve all but that test, or approve all and then fix up if needed?

Reply:
I am good with merging the more lenient version of this test and discussing on the separate issue in the meantime. 

I did approve the PR, see above "drott approved these changes 23 hours ago", but it seems like that doesn't count 😢.

"Review required
At least one approved review is required by reviewers with write access. "

Would I be eligible for write access, who could configure it for me?

Reply:
> Would I be eligible for write access, who could configure it for me?

I just did, https://github.com/w3c/web-platform-tests/pull/9525 but now that tiny edit needs a reviewer ...